tsconfig.build.json

```diff
@@ -8,7 +8,7 @@
     "sourceMap": true,
     "rootDir": "./",
     "strict": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
I thought we wanted to avoid using declarations with the implied any type

Reply:
Yes, the problem is productapi is still in js and basketapi is in ts, when I exported a function from productapi for basketapi to import, typsescript does not like it. This is a hack to make it to work without converting the whole productapi to typescript, otherwise I will have to repeat the whole getPrices() in priceHelpers.js in basketapi.

Reply:
@coopaq @blittle any thoughts on this one?

Reply:
After converting all productapi service to typescript, we will turn this back on.